package.json

```diff
@@ -46,14 +46,14 @@
     "govuk-frontend": "^5.7.1"
   },
   "devDependencies": {
-    "sass": "1.77.6",
     "@babel/preset-env": "^7.21.5",
+    "@playwright/browser-chromium": "^1.49.1",
```

Comment:
this dependency means that playwright chromium browser will be downloaded on install